network-api/networkapi/wagtailpages/migrations/0071_datawrapper_block.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.1.11 on 2022-02-08 15:40
+
+import django.core.validators
+from django.db import migrations
+import networkapi.wagtailpages.pagemodels.customblocks.articles
+import wagtail.contrib.table_block.blocks
+import wagtail.core.blocks
+import wagtail.core.blocks.static_block
+import wagtail.core.fields
+import wagtail.embeds.blocks
+import wagtail.images.blocks
+import wagtailmedia.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailpages', '0070_auto_20220126_2241'),
```

Comment:
File becomes `0072_datawrapper_block.py` and dependency becomes `0071_auto_20220208_0037`